src/bandersnatch/filter.py

```diff
@@ -188,3 +214,15 @@ def filter_release_plugins() -> Iterable[Filter]:
         List of objects derived from the bandersnatch.filter.Filter class
     """
     return load_filter_plugins(RELEASE_PLUGIN_RESOURCE)
+
+
+def filter_release_file_plugins() -> Iterable[Filter]:
```

Comment:
Probably want Sequence[Filter] here as a str is iterable ...
If interested, types are based on: https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes